src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Mashine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck  почав працювати.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Truck закінчив працювати.");

```

Comment:
The messages in the `doWork` and `stopWork` methods should be in English. Consider changing them to something like "Truck started working." and "Truck stopped working." to meet the checklist requirement of using English and making messages informative.